Pod/Core/StepDefiner.swift

```diff
@@ -114,52 +91,28 @@ open class StepDefiner: NSObject {
      
      Don't pass anything for file: or path: - these will be automagically filled out for you. Use it like this:
      
-         step("Some (regular|irregular) expression with a second capture group here (.*)") { (match1: String, match2: String) in
+         step("Some (regular|irregular) expression with a second capture group here (.*)") { (match1: String, match2: Int) in
              ... some function ...
          }
      
      - parameter expression: The expression to match against
-     - parameter f2s: The step definition to be run, passing in the first two capture groups from the expression
-     */
-    open func step(_ expression: String, file: String = #file, line: Int = #line, f2s: @escaping (String, String)->()) {
-        self.test.addStep(expression, file: file, line: line) { (matches: [String]) in
-            
-            guard matches.count >= 2 else {
-                XCTFail("Expected at least 2 matches, found \(matches.count) instead, from \"\(expression)\"")
-                return
-            }
-            
-            f2s(matches[0], matches[1])
-        }
-    }
-    
-    /**
-     If you only want to match the first parameters as integers, this will help make your code nicer
-     
-     Don't pass anything for file: or path: - these will be automagically filled out for you. Use it like this:
-     
-     step("Some (regular|irregular) expression with a second capture group here (.*)") { (match1: Int, match2: Int) in
-     ... some function ...
-     }
-     
-     - parameter expression: The expression to match against
-     - parameter f2i: The step definition to be run, passing in the first two capture groups from the expression
+     - parameter f2: The step definition to be run, passing in the first two capture groups from the expression
      */
-    open func step(_ expression: String, file: String = #file, line: Int = #line, f2i: @escaping (Int, Int)->()) {
+    open func step<T: FromStringable, U: FromStringable>(_ expression: String, file: String = #file, line: Int = #line, f2: @escaping (T, U)->()) {
         self.test.addStep(expression, file: file, line: line) { (matches: [String]) in
             
             guard matches.count >= 2 else {
                 XCTFail("Expected at least 2 matches, found \(matches.count) instead, from \"\(expression)\"")
                 return
             }
             
-            guard let i1 = Int(matches[0]),
-                let i2 = Int(matches[1]) else {
+            guard let i1 = T.fromString(matches[0]),
+                let i2 = U.fromString(matches[1]) else {
                     XCTFail("Could not convert matches (\(matches[0]) and \(matches[1])) to integers, from \"\(expression)\"")
```

Comment:
This error message should be updated to show that we're not matching to integers anymore, but to `T.self` (and `U.self`)